scripts/core/editor3/service.ts

```diff
@@ -149,7 +149,7 @@ export class EditorService {
         }
 
         if (typeof spellcheck !== 'undefined') {
-            spellcheckerStores.map((s) => s.dispatch(action.setSpellcheckerLanguage()));
+            spellcheckerStores.map((s) => s.dispatch(action.setSpellcheckerLanguage(language)));
```

Comment:
Change `map` to `forEach`. There's no point(except to confuse the developer looking at the code) in using `map` when you don't use its result.